src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java

```diff
@@ -465,28 +468,30 @@ public String getSQLKeywords() throws SQLException {
 	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
```

Comment:
Now that this function is implemented we can probably remove `@NotImplemented`?

Reply:
Same in the other methods.

Reply:
sure. done